autoload/nvimgdb/ui.vim

```diff
@@ -41,6 +42,7 @@ function! s:DefineCommands()
   command! GdbStep call GdbSend('s')
   command! GdbFinish call GdbSend('finish')
   command! GdbFrameUp call GdbSend('up')
+  command! GdbFrame call GdbSend('f')
```

Comment:
How would it work in the other backends? Care to write a basic test case for this functionality?